os_migrate/plugins/module_utils/reference.py

```diff
@@ -2,6 +2,54 @@
 __metaclass__ = type
 
 
+def network_name(conn, id_, required=True):
```

Comment:
We might then replace calls like https://github.com/os-migrate/os-migrate/blob/b183976cceb0bc48207aa9214b1cea9dab3e2a1f/os_migrate/plugins/modules/export_subnet.py#L79

Reply:
This method only returns name if the ID is non-null, it does not return the whole network. It is to DRY the "if not None, give me <something>" use cases, which otherwise result in boilerplate in the calling code.